src/Wallabag/CoreBundle/Resources/translations/messages.en.yml

```diff
@@ -323,7 +323,7 @@ howto:
         go_import: Go to import
         go_developers: Go to developers
         go_howto: Go to howto (this page!)
-        go_logout: Logout
+        go_logout: Log out
```

Comment:
If I understand https://whitelightconcepts.com/login-vs-log-in-logout-vs-log-out/ correctly, I think it should stay in one word.

Reply:
How is it used in wallabag? You click the button and what happens?
"Login" makes sense for talking about the process of logging in, the portal as it were. Clicking to do something however, is two words. I can't grasp logging out being more complex than clicking the "Log out"-button…?

Reply:
There's no intermediate page when you click on this link, you're directly logged out. 'Log out' seems more accurate here.

Reply:
@tcitworld It is indicative of an action occuring as a result of clicking it, so "Log out" applies.
You got to the logout page (or "Log out"-page) and click "Log out" to perform an action.